azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor/EventData.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.monitor;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+import com.microsoft.azure.management.apigeneration.LangMethodDefinition;
+import com.microsoft.azure.management.monitor.implementation.EventDataInner;
+import org.joda.time.DateTime;
+
+import java.util.Map;
+
+/**
+ * The Azure event log entries are of type EventData.
+ */
+@LangDefinition(ContainerName = "/Microsoft.Azure.Management.Monitor.Fluent.Models")
+public interface EventData {
```

Comment:
`extends HasInner<EventDataInner>` so that inner() exposed by `EventDataImpl` (via WrapperImpl) is visible to user